### ISSUE TITLE
Embedded Player Failure - Cannot find module 'babel-preset-env'

### ISSUE THREAD

**Issue by bkuhlmann** [asciinema/asciinema-server#355]
## Overview

Started seeing the following error today:

<details>
  <summary>Full Error</summary>
``` javascript
throw new Error("Module build failed (from ./node_modules/babel-loader/lib/index.js):\nError: Cannot find module 'babel-preset-env' from '/Users/sickill/projects/asciinema/asciinema-server/assets'\n- Did you mean \"@babel/env\"?\n    at Function.resolveSync [as sync] (/Users/sickill/projects/asciinema/asciinema-server/assets/node_modules/resolve/lib/sync.js:89:15)\n    at resolveStandardizedName (/Users/sickill/projects/asciinema/asciinema-server/assets/node_modules/@babel/core/lib/config/files/plugins.js:101:31)\n    at resolvePreset (/Users/sickill/projects/asciinema/asciinema-server/assets/node_modules/@babel/core/lib/config/files/plugins.js:58:10)\n    at loadPreset (/Users/sickill/projects/asciinema/asciinema-server/assets/node_modules/@babel/core/lib/config/files/plugins.js:77:20)\n    at createDescriptor (/Users/sickill/projects/asciinema/asciinema-server/assets/node_modules/@babel/core/lib/config/config-descriptors.js:154:9)\n    at /Users/sickill/projects/asciinema/asciinema-server/assets/node_modules/@babel/core/lib/config/config-descriptors.js:109:50\n    at Array.map (<anonymous>)\n    at createDescriptors (/Users/sickill/projects/asciinema/asciinema-server/assets/node_modules/@babel/core/lib/config/config-descriptors.js:109:29)\n    at createPresetDescriptors (/Users/sickill/projects/asciinema/asciinema-server/assets/node_modules/@babel/core/lib/config/config-descriptors.js:101:10)\n    at presets (/Users/sickill/projects/asciinema/asciinema-server/assets/node_modules/@babel/core/lib/config/config-descriptors.js:47:19)\n    at mergeChainOpts (/Users/sickill/projects/asciinema/asciinema-server/assets/node_modules/@babel/core/lib/config/config-chain.js:320:26)\n    at /Users/sickill/projects/asciinema/asciinema-server/assets/node_modules/@babel/core/lib/config/config-chain.js:283:7\n    at Generator.next (<anonymous>)\n    at buildRootChain (/Users/sickill/projects/asciinema/asciinema-server/assets/node_modules/@babel/core/lib/config/config-chain.js:120:29)\n    at buildRootChain.next (<anonymous>)\n    at loadPrivatePartialConfig (/Users/sickill/projects/asciinema/asciinema-server/assets/node_modules/@babel/core/lib/config/partial.js:95:62)\n    at loadPrivatePartialConfig.next (<anonymous>)\n    at Function.<anonymous> (/Users/sickill/projects/asciinema/asciinema-server/assets/node_modules/@babel/core/lib/config/partial.js:120:25)\n    at Generator.next (<anonymous>)\n    at evaluateSync (/Users/sickill/projects/asciinema/asciinema-server/assets/node_modules/gensync/index.js:244:28)\n    at Function.sync (/Users/sickill/projects/asciinema/asciinema-server/assets/node_modules/gensync/index.js:84:14)\n    at Object.<anonymous> (/Users/sickill/projects/asciinema/asciinema-server/assets/node_modules/@babel/core/lib/config/index.js:41:61)\n    at Object.<anonymous> (/Users/sickill/projects/asciinema/asciinema-server/assets/node_modules/babel-loader/lib/index.js:151:26)\n    at Generator.next (<anonymous>)\n    at asyncGeneratorStep (/Users/sickill/projects/asciinema/asciinema-server/assets/node_modules/babel-loader/lib/index.js:3:103)\n    at _next (/Users/sickill/projects/asciinema/asciinema-server/assets/node_modules/babel-loader/lib/index.js:5:194)\n    at /Users/sickill/projects/asciinema/asciinema-server/assets/node_modules/babel-loader/lib/index.js:5:364\n    at new Promise (<anonymous>)\n    at Object.<anonymous> (/Users/sickill/projects/asciinema/asciinema-server/assets/node_modules/babel-loader/lib/index.js:5:97)\n    at Object.loader (/Users/sickill/projects/asciinema/asciinema-server/assets/node_modules/babel-loader/lib/index.js:64:18)\n    at Object.<anonymous> (/Users/sickill/projects/asciinema/asciinema-server/assets/node_modules/babel-loader/lib/index.js:59:12)");
```
</details>

## Steps to Recreate

1. Visit this [page](https://www.alchemists.io/screencasts/git_rebase_exec) and notice the embedded player never loads.
1. If you open up your browser inspector, you'll see the error, posted above, in your console.

:bulb: This error does not happen on the [main ascinema server](https://asciinema.org/a/333725), though.

## Screenshots/Screencasts

![screen-oGSce9JO@2x](https://user-images.githubusercontent.com/50245/82819291-61e89100-9e5d-11ea-886a-6f0df56a0f2f.png)

**Comment by bkuhlmann** [comment]
:information_source: Seeing others post about this in the [asciinema forum](https://discourse.asciinema.org/t/embed-player-dont-work/365) as well.

**Comment by iblancasa** [comment]
Same problem here :'(

**Comment by sickill** [comment]
Good catch guys. I'm looking into this right now 🤞

**Comment by sickill** [comment]
Found and fixed the problem. Apparently my local development artifacts got their way into the docker image I deployed to asciinema.org 2 days ago. Just [made sure this doesn't happen again](https://github.com/asciinema/asciinema-server/commit/9412c70f885fc21b6487a413976285a23717e10f), rebuilt it and re-deployed. Looks like it's working fine now for your case @bkuhlmann, as well as for the case reported on the forum.
